src/txrebroadcast.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_TXREBROADCAST_H
+#define BITCOIN_TXREBROADCAST_H
+
+#include <policy/feerate.h>
+#include <tuple>
+#include <txmempool.h>
+#include <util/hasher.h>
+#include <util/time.h>
+#include <validation.h>
+
+#include <boost/multi_index/hashed_index.hpp>
+#include <boost/multi_index/member.hpp>
+#include <boost/multi_index/ordered_index.hpp>
+#include <boost/multi_index_container.hpp>
+
+struct TxIds {
+    TxIds(uint256 txid, uint256 wtxid) : m_txid(txid), m_wtxid(wtxid) {}
+
+    const uint256 m_txid;
+    const uint256 m_wtxid;
+};
+
+struct RebroadcastEntry {
+    RebroadcastEntry(std::chrono::microseconds now_time, uint256 wtxid)
+        : m_last_attempt(now_time),
+          m_wtxid(wtxid),
+          m_count(1) {}
+
+    std::chrono::microseconds m_last_attempt;
+    const uint256 m_wtxid;
+    int m_count;
```

Comment:
Prefer default initialization when a member will always be initialized to the same value.

```suggestion
    int m_count{1};
```

And remove `m_count` from the initializer list.

Reply:
done